lib/beaker/host/unix/pkg.rb

```diff
@@ -417,8 +417,14 @@ def puppet_agent_dev_package_info( puppet_collection = nil, puppet_agent_version
         puppet_collection, puppet_agent_version, opts )
     when /^(sles|aix|el|centos|oracle|redhat|scientific)$/
       variant = 'el' if variant.match(/(?:el|centos|oracle|redhat|scientific)/)
-      arch = 'ppc' if variant == 'aix' && arch == 'power'
-      version = '7.1' if variant == 'aix' && version == '7.2'
+      if variant == 'aix'
+        arch = 'ppc' if arch == 'power'
+        if %w(PC1 puppet5).include? puppet_collection
+          version = '7.1' if version == '7.2'
+        else
+          version = '6.1'
+        end
+      end
       release_path_end = "#{variant}/#{version}/#{puppet_collection}/#{arch}"
       release_file = "puppet-agent-#{puppet_agent_version}-1.#{variant}#{version}.#{arch}.rpm"
```

Comment:
I actually think these paths will be incorrect -- it looks like all shipped builds still use 'PC1' in their URLs, regardless of the collection. I'll block this for now.

Reply:
Ok, I've updated this to rely on version numbers instead of collection names. This is now blocked on an update of the reported version in puppet-agent#master to 5.99.z -- I'll unblock as soon as that happens.